### PR TITLE
Altered the existing Green JS Logo SVG

### LINKED ISSUE #23: Alter the existing Green JS Logo SVG
Alter the existing green JS Logo to the site assets. 

> [Read more about the Node.js Website redesign](https://github.com/nodejs/nodejs.org/discussions/5131)

image

> from the [Figma design (direct link)](https://www.figma.com/file/pu1vZPqNIM7BePd6W8APA5/Node.js?type=design&node-id=337-7893&mode=design&t=ZmAh8itCUQCRhlFv-4)

> displaying both. This issue only altersthe green one.

---

### Before You Start...

- Use the [Contributor's Guide](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) to learn how to [setup your environment](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md#getting-started), run [available commands](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md#cli-commands), and construct [proper commits](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md#commit-guidelines).
- Review the [technologies used](https://github.com/nodejs/nodejs.org/blob/main/COLLABORATOR_GUIDE.md#technologies-used-in-the-website) when building the site, and [why they were chosen](https://github.com/nodejs/nodejs.org/blob/main/COLLABORATOR_GUIDE.md#remarks-on-technologies-used).
- Review the [component creation guide](https://github.com/nodejs/nodejs.org/blob/main/COLLABORATOR_GUIDE.md#creating-components) to follow our standards.

Make sure that `npm ci` completes for you in a terminal pointing to your fork. If it doesn't or you aren't sure, ask a mentor.

---

#### Where to add the file

Site assets are found at https://github.com/nodejs/nodejs.org/tree/main/public/static/images/logos. 

- Alter this file in this directory _on your fork_

#### Adding the file

This file is an SVG, which means it is represented as text.

Find the file in the `public/static/images/logos` directory, called `js-green.svg`

Overwrite the existing content with a paste of the below contents into the existing file and save.

```
<svg width="71" height="80" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#clip0_337_7891)" fill="#5FA04E"><path d="M35.625 79.5c-1.081 0-2.09-.288-3.028-.792l-9.59-5.686c-1.442-.792-.721-1.08-.289-1.224 1.947-.648 2.308-.792 4.327-1.944.216-.144.504-.072.72.072l7.356 4.391c.288.144.649.144.865 0l28.77-16.628c.289-.144.433-.431.433-.791V23.714c0-.36-.144-.648-.432-.792L35.986 6.366c-.288-.144-.65-.144-.865 0L6.35 22.922c-.29.144-.434.504-.434.792v33.184c0 .287.145.647.433.791l7.86 4.535c4.254 2.16 6.922-.36 6.922-2.879V26.593c0-.432.36-.864.865-.864h3.678c.432 0 .865.36.865.864v32.752c0 5.687-3.1 8.998-8.509 8.998-1.658 0-2.956 0-6.633-1.8l-7.572-4.319A6.073 6.073 0 0 1 .798 56.97V23.786a6.073 6.073 0 0 1 3.028-5.255l28.77-16.628c1.804-1.008 4.255-1.008 6.058 0l28.77 16.628a6.073 6.073 0 0 1 3.029 5.255V56.97a6.073 6.073 0 0 1-3.029 5.254l-28.77 16.628c-.865.36-1.947.648-3.029.648Z"/><path d="M44.567 56.682c-12.62 0-15.215-5.759-15.215-10.654 0-.432.36-.864.865-.864h3.75c.433 0 .793.288.793.72.577 3.815 2.235 5.687 9.879 5.687 6.057 0 8.652-1.368 8.652-4.607 0-1.871-.72-3.24-10.167-4.175-7.86-.792-12.762-2.52-12.762-8.782 0-5.83 4.903-9.285 13.123-9.285 9.23 0 13.772 3.167 14.35 10.077 0 .216-.073.432-.217.648-.144.144-.36.288-.577.288h-3.822a.844.844 0 0 1-.793-.648c-.865-3.96-3.1-5.255-9.013-5.255-6.634 0-7.427 2.304-7.427 4.031 0 2.088.937 2.736 9.879 3.887 8.869 1.152 13.05 2.808 13.05 8.998 0 6.335-5.263 9.934-14.348 9.934Z"/></g><defs><clipPath id="clip0_337_7891"><path fill="#fff" d="M0 .5h71v79H0z"/></clipPath></defs></svg>
```

#### Adding a Storybook story to preview the file

Use [Storybook](https://storybook.js.org/) to preview the new logo.

Add a file called `js-logos.stories.tsx` to the `components/__design__` directory. If this file already exists, open it instead.

We want to create a file that looks like this, eventually, for this and the white JS logo (a separate story)

```tsx
import Image from 'next/image';
import type { Meta as MetaObj, StoryObj } from '@storybook/react';

export const JSLogos: StoryObj = {};

export default {
  title: 'Design System/Logos',
  component: () => {
    return (
    <div>
      <div>
        <Image
            src={`/static/images/logos/js-white.svg`}
            alt="Node.js"
            width={30}
            height={30}
          />
      </div>
      <div>
        <Image
            src={`/static/images/logos/js-green.svg`}
            alt="Node.js"
            width={30}
            height={30}
          />
      </div>
    </div>
  )},
} as MetaObj;
```

> Only add the `Image` for the logo you added.

#### Running Storybook

In a terminal, run `npm run storybook`.

Then, in a browser, go to http://localhost:6006/?path=/story/design-system-logos--js-logos and you should see the file you added.

<details><summary>Preview</summary>

</details>

#### Submitting the changes

[Submit a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) back to the main repository.

### PR DESCRIPTION
**Pull request by Pooja-Codes**
## Description

Altered the JS Green SVG logo file and added a Storybook story to preview the file.

## Validation

Varified via `npm run storybook` and previewed the altered JS Green SVG logo.

## Related Issues

Fixes [#23](https://github.com/bmuenzenmeyer/nodejs.org/issues/23)

### Check List

Followed the below checklist.

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [x] I have run `npx turbo format` to ensure the code follows the style guide.
- [x] I have run `npx turbo test` to check if all tests are passing.
- [x] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

File: components/__design__/js-logos.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+export const JSLogos: StoryObj = {};
+
+export default {
+  title: 'Design System/Logos',
+  component: () => {
+    return (
+      <div>
+        <div>
+          <Image
+            src={`/static/images/logos/js-white.svg`}
```
Comment (ovflowd) [inline components/__design__/js-logos.stories.tsx:13]: I don't think the `js-white.svg` file exists yet? 

I believe this PR requires https://github.com/nodejs/nodejs.org/pull/5858 to be merged first, so let me approve that other PR first and merge, so you can rebase yours :)

**Review by ovflowd — APPROVED** [review]
Thank you for first time contribution! Really good stuff! ✨ 🎉 

I appreciate the time and effort put here!

File: components/__design__/js-logos.stories.tsx (hunk 1 above)
Comment (Pooja-Codes) [reply components/__design__/js-logos.stories.tsx:13]: I agree! PR #5858 has been merged. Please review again :)

**Comment by ovflowd** [comment]
Hey @Pooja-Codes 👋 can you update your PR to instead of creating a new file, update https://github.com/nodejs/nodejs.org/blob/main/components/__design__/logos.stories.tsx this existing file? Thank you!

**Comment by Pooja-Codes** [comment]
There are two conflicting issues [#23](https://github.com/bmuenzenmeyer/nodejs.org/issues/23) (Alter the file) and [#5835](https://github.com/nodejs/nodejs.org/issues/5835) (Delete the file) for `js-green.svg` file. 

Now, PR [#5877](https://github.com/nodejs/nodejs.org/pull/5877) (Delete the file) is already merged and it makes this PR (Alter the file) baseless! 

Can you guide me on how to fix this? @ovflowd

**Comment by Pooja-Codes** [comment]
> Hey @Pooja-Codes 👋 can you update your PR to instead of creating a new file, update https://github.com/nodejs/nodejs.org/blob/main/components/__design__/logos.stories.tsx this existing file? Thank you!

I'm not getting any conflict with `logos.stories.tsx` after syncing with the mainstream!

**Comment by bmuenzenmeyer** [comment]
@Pooja-Codes sorry for the confusion - that was my mistake in trying to prepare issues for the event. we want the file back per https://github.com/bmuenzenmeyer/nodejs.org/issues/23

**Comment by Pooja-Codes** [comment]
> @Pooja-Codes sorry for the confusion - that was my mistake in trying to prepare issues for the event. we want the file back per [bmuenzenmeyer#23](https://github.com/bmuenzenmeyer/nodejs.org/issues/23)

@bmuenzenmeyer It's fine! 😅
I'm requesting @ovflowd  to review it again! Thank you!😊

**Comment by bmuenzenmeyer** [comment]
these stories have conflicts that require resolution - i'll doing that hot as we merge them in

**Pull request merged**